README.md

```diff
@@ -121,6 +121,7 @@ For every single NPM package, we create a record in the Algolia index. The resul
     ts: 'definitely-typed', // definitely-typed | included | false
     definitelyTyped: '@types/babel__core',
   },
+  moduleType: 'cjs', // esm | cjs | unknown
```

Comment:
I would make that an array, some package build for esm and cjs and browser and umd...

Reply:
Good idea, umd or browser build is too hard to distinguish though. Can check for the browser field, but is only necessary if package is needing  environment-specific code. Just cjs & esm where I’m 100% sure for now